cmd/influxd/run.go

```diff
@@ -231,52 +234,76 @@ func parseConfig(path, hostname string) *Config {
 }
 
 // creates and initializes a broker.
-func openBroker(path string, u *url.URL, initializing bool, joinURLs []*url.URL, w io.Writer) *influxdb.Broker {
+func openBroker(path string, u url.URL, initializing bool, joinURLs []url.URL, w io.Writer, raftTracing bool) (*influxdb.Broker, *raft.Log) {
+	// Create raft log.
+	l := raft.NewLog()
```

Comment:
It makes `messaging` much easier to test when we can control the interface to the Raft log. I'm trying to break pieces out into more testable chunks.